Lib/json/tool.py

```diff
@@ -16,6 +16,18 @@
 import sys
 
 
+def parse_indent(indent):
+    """Parse the argparse indent argument."""
+    if indent == 'None':
+        return None
+    if indent == r'\t':
+        return '\t'
```

Comment:
I don't like this either. You can use external commands like `unexpand` for converting spaces to tabs.